etc/profile-m-z/nodejs-common.profile

```diff
@@ -67,6 +67,7 @@ include disable-xdg.inc
 whitelist /usr/share/doc/node
 whitelist /usr/share/nvm
 whitelist /usr/share/systemtap/tapset/node.stp
+include build-systems-common.profile
```

Comment:
- This should go to the end of the file, see https://github.com/netblue30/firejail/blob/master/etc/templates/redirect_alias-profile.template.
- All command set by `build-systems-common.profile` should be removed here.
- All command unintentional introduced by this must be `ignore`d.

> > It can also, instead of including `build-systems-common.profile` just whitelist `/usr/share/pkgconfig`.

Likely easier than the above.